packages/components/src/radio-control/README.md

```diff
@@ -1,11 +1,59 @@
 # RadioControl
 
-RadioControl component is used to generate radio input fields.
+Use radio buttons when you want users to select one option from a set, and you want to show them all the available options at once.
 
+![Selected and unselected radio buttons](./radio.png)
+Selected and unselected radio buttons
+
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
+
+# Design guidelines
```

Comment:
It's such a small thing but it would be good to get an accessibility review on this; I know in the past it was suggested to keep one `<h1>` per page, but that may have changed. I know some a11y tech uses heading levels to assist navigation so it's important to get this right.